test/galaxy_selenium/has_driver.py

```diff
@@ -14,6 +15,7 @@
 
 
 class HasDriver:
+    TimeoutException = TimeoutException
```

Comment:
I have avoided selenium imports in ``test/galaxy_selenium/navigates_galaxy.py`` and all of the ``test/selenium_tests`` files - this was required to to continue that. I didn't necessarily set out to do be so encapsulated - but it is a nice property to have now.

If you insist I will drop this and add a direct import of this class from ``navigates_galaxy.py``.